zsh-syntax-highlighting.zsh

```diff
@@ -188,7 +188,7 @@ _zsh_highlight_apply_zle_highlight() {
   integer first="$3" second="$4"
 
   # read the relevant entry from zle_highlight
-  local region="${zle_highlight[(r)${entry}:*]}"
+  local region="${zle_highlight[(r)${entry}:*]-}"
```

Comment:
Independent/preexisting issue: needs to be ${(b)entry}.

Reply:
That should be it's own commit though.

Reply:
Agreed.